web/src/main.js

```diff
@@ -794,6 +794,15 @@ if (!getState().application.isMobile) {
   });
 }
 
+// Production/Consumption
```

Comment:
can you also remove the toggle that I coded in the bottom of main.js (that binds to `window`?)

Reply:
done